install_config/install/advanced_install.adoc

```diff
@@ -842,15 +844,66 @@ openshift_cockpit_deployer_prefix='registry.example.com/openshift3/'
 openshift_cockpit_deployer_version='3.9.3'
 ----
 
+
+[[advanced-install-glusterfs-persistent-storage]]
+=== Configuring {gluster} Persistent Storage
+
+include::install_config/persistent_storage/topics/glusterfs_intro.adoc[]
+
+[[advanced-install-containerized-glusterfs-persistent-storage]]
+==== Configuring {gluster-native}
+
+[IMPORTANT]
+====
+See
+xref:../../install_config/install/prerequisites.adoc#prereq-containerized-glusterfs-considerations[{gluster-native} Considerations]
+for specific host preparations and prerequisites.
+====
+
+include::install_config/persistent_storage/topics/glusterfs_example_basic.adoc[]
+
+[[advanced-install-external-glusterfs-persistent-storage]]
+==== Configuring {gluster-external}
+
+include::install_config/persistent_storage/topics/glusterfs_example_basic_external.adoc[]
+
+[[advanced-install-registry]]
+=== Configuring an OpenShift Container Registry
```

Comment:
All instances of OpenShift should be {product-title}. This will render correctly, depending on the distro and branch build.

Reply:
I don't think this applies here. The name of the product in question is "OpenShift Container Registry" regardless of whether it's Enterprise or Origin. I'm using this as my reference: https://github.com/openshift/openshift-docs/blob/master/install_config/install/advanced_install.adoc#advanced-install-configuring-openshift-container-registry

Reply:
I don't think this applies here. The name of the product in question is "OpenShift Container Registry" regardless of whether it's Enterprise or Origin. There are several instances of that name being used across the repo.